docs/source/additional/privacy-strategy.rst

```diff
@@ -0,0 +1,133 @@
+
+This part of the documentation will be updated regularly to reflect the latest available research.
+
+Privacy Enhancing Technologies.
+-------------------------------
+
+We touch on a **few** of the main technologies that are making collaborative data sharing possible today in ways that can be considered more secure.
+
+**Federated Learning**:
+FL allows Machine Learning models to be sent to servers where they can train and test on data without having to ever move the data from its original location. This idea however is not restricted to machine learning and is then referred to as Federated Analytics. The Substra software allows for both Federated Learning and Federated Analytics.
+
+**Secure Enclaves**: 
+These are hardware-based features that provide an isolated environment to store and process data. A secure enclave is essentially an ultra secure space within a larger secure system. Although they are excellent for safely storing data, the privacy is hardware dependent and places trust in a physical chip as opposed to encryption.
+
+**Differential Privacy**:
+This approach adds noise throughout processing to prevent inferring item-specific information. This comes with a tradeoff between privacy and performance though, as the addition of noise might cause models to be less accurate. DP provides theoretical bounds on how hard it is to infer the presence of a given individual in a dataset when it is used.
+
+**Secure Multi-Party Computation (SMPC)**: 
+This method enables partners to jointly analyze data through the use of cryptography. Users can conduct analysis together but do not have to reveal exact data points to each other. Secure Aggregation is a form of SMPC where the Federated aggregation of the models is done in such a way that the individual models are not disclosed, but the average of the models is still computed.
+
+Privacy is a hard problem which is why it is usually recommended to combine several PETs when working on projects involving highly sensitive data. Substra enables Federated Learning for your projects, while its flexibility makes it fully compatible with other privacy-enhancing libraries, as shown in the (MELLODDY project)[https://ojs.aaai.org/index.php/AAAI/article/view/26847].
+
+Security risks and threat models
+--------------------------------
+
+When performing FL or using any privacy enhancing technologies, it's important to understand the exact guarantees provided by a technology and the possible threats. Federated networks are complex systems. In order to understand the risks regarding data privacy, it is essential to explicitly consider the scenarios you are protecting against and the hypotheses you are making. 
```

Comment:
Do we capitalise Privacy-Enhancing Technologies everywhere or not? 
Same question for the hyphen

Reply:
Capitalized, no hyphen for consistency